wiki/Beatmap/Pattern/Mania/Trills/en.md

```diff
@@ -0,0 +1,29 @@
+---
+stub: true
+tags:
+  - Trills
+---
+
+# Trills
+
+A **Trill** is two or more consecutive notes that alternate between two different columns. 
+
+![](img/trill.png)
+
+**Trills** are often characterized based on whether they require two hand movements or one. These are known as **one-handed trills** and **two-handed trills**.
+
+## Chordtrills, Jumptrills and Split-Trills
+
+**Chordtrills** make use of two consecutive alternating chords instead of singular notes. 
+
+![](img/chordtrill.png)
+
+There are two main types of **chordtrills**. There are **chordtrills** which require each chord to be played with one hand known as **jumptrills**.
+
+![](img/jumptrill.png)
+
+The other type, **split-trills**, require each chord to be played with two hands.
+
+![](img/splittrill.png)
+
+For 4-key osu!mania, these are the only types of **chordtrills**. As such, the term **chordtrill** is more likely to be used in higher key modes.
```

Comment:
Would maybe add somewhere that a chordtrill or any kind of trill that uses chords are, in theory, just multiple trills at once. This is because we define a Trill as something that alternates between two different columns

Reply:
What do you think of this? 
```suggestion
**Chordtrills** make use of two consecutive alternating chords instead of singular notes. They can also be imagined as multiple trills occurring at once within the same snap intervals of each other.

![](img/chordtrill.png)

There are two main types of **chordtrills**. There are **chordtrills** which require each chord to be played with one hand known as **jumptrills**.

![](img/jumptrill.png)

The other type, **split-trills**, require each chord to be played with two hands.

![](img/splittrill.png)

For 4-key osu!mania, these are the only types of **chordtrills**. As such, the term **chordtrill** is more likely to be used in higher key modes.
```

Reply:
I like it